osgtest/tests/test_401_scitokens.py

```diff
@@ -21,13 +21,13 @@ def test_01_request_condor_write_scitoken(self):
         )
 
     def test_02_request_xrootd_scitokens(self):
-        self.skip_ok_unless("SCITOKENS" in core.config.get('xrootd.security', []),
+        self.skip_ok_unless("SCITOKENS" in core.config.get('xrootd.security', set()),
                             "Not using SciTokens for XRootD")
         try:
             credentials.request_scitoken(
                 "xrootd",
                 subject=core.options.username,
-                scope="write:/",
+                scope=f"read:/ write:/{core.config['xrootd.user_subdir']}",
```

Comment:
Should we also restrict read on this token or is restricted access of VO area restricted by the authfile?

Reply:
Let's restrict read access as well. I think the authfile is ignored for tokens.